### PR TITLE
Ins 387 seed observations

### PR DESCRIPTION
**Pull request by ghost**
- Added seed script to import data from csv file
- Added a new table to track the seed activities
- Updated the Form Framework and added a new section for `batchImport`
- Added `None` option for soil texture and specific use code table

### CODE REVIEW

File: FormFrameworkREADME.md [hunk 1]
```diff
@@ -102,6 +102,34 @@ The starting point for a new form is the schema's .yaml file, which specifies th
...
+            radius: radius
+  ```
+  Some options take precendence over the other, the script first process the `allColumnsExcept`, then the `mapper` is executed and finally it groups the fields mentioned in `groupField`. Fill in the options as mentioned in the example above. 
+
+  - `fileName` - The name of the CSV file to load
+  - `environments` - List of environments where the import should happen - dev, test, prod
+  - `allColumnsExcept` - The fields mentioned here are not processed during import
+  - `mapper` - The key/value pairs to map the existing column value to a different field
+  - `groupFields` - Group the `fields` mentioned with a new `key`
+
```
Comment (andrea-williams) [inline FormFrameworkREADME.md:131]: thank you for adding this to the readme Raj! 🙌

**Review by andrea-williams — DISMISSED** [review]
✅ Jenkins build passes

File: api/api_sources/resources/csv/SpecificUseCode.csv [hunk 2]
```diff
@@ -21,3 +21,4 @@ MI,Mine site
 MT,Mine tailings
 BS,Burn scar
 IS,Industrial site
+NA,None
```
Comment (fwpushan) [inline api/api_sources/resources/csv/SpecificUseCode.csv:24]: Currently,  we are running the migration progressively, so make sure new values should be migrated as fresh new migration.

File: api/api_sources/resources/csv/SoilTextureCode.csv [hunk 3]
```diff
@@ -3,3 +3,4 @@ C  ,Coarse (Sand / cobbles / gravel - water drains quickly)
 F  ,Fine (Clay - compact soil that holds water)
 M  ,Medium (Loamy / silty soil - water takes longer to drain)
 O  ,Organic (Dark soil with organic material)
+N  ,None
```
Comment (fwpushan) [inline api/api_sources/resources/csv/SoilTextureCode.csv:6]: Currently,  we are running the migration progressively, so make sure new values should be migrated as fresh new migration.

File: api/api_sources/sources/database/migrations/1588201708197-SeedSchema.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        this.log('[START]', 'UP');
+        await queryRunner.query(this.seedSchema.migrationSQL);
+        this.log('[END]', 'UP');
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+        this.log('[START]', 'DOWN');
+        await queryRunner.query(this.seedSchema.dropTable());
```
Comment (fwpushan) [inline api/api_sources/sources/database/migrations/1588201708197-SeedSchema.ts:22]: Use method _**await this.runQuerySqlFiles(this.upMigrations(), queryRunner);**_

File: api/api_sources/sources/libs/core-database/seed.db.ts [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+                }
+            }
+
+            mappedData.push(dataObj);
+        }
+
+        return mappedData;
+    }
+
+    public static createSpaceGeomObject(data: any) {
```
Comment (fwpushan) [inline api/api_sources/sources/libs/core-database/seed.db.ts:191]: For data mapping and schema specific handling, try to use the delegate model.

**Review by fwpushan — CHANGES_REQUESTED** [review]
Good Job so far, feel free to discuss it with me.

**Review by fwpushan — CHANGES_REQUESTED** [review]
You should for dev env also.

**Review by fwpushan — APPROVED** [review]
We must get clarification on subHex id calculation and re-seeding.

**Pull request merged**